content/javascript/concepts/date-methods/getDate.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: ".getDate()"
+Subjects:
+  - "Web Development"
+  - "Computer Science"
+Tags:
+  - "Date"
+  - "Methods"
+Catalog Content:
+  - "https://www.codecademy.com/learn/paths/web-development"
+---
+
+## Definition
+
+Get the date of the month according to the local time.
+
+## Syntax
+
+```js
+Date.getDate();
```

Comment:
This should not be called on the `Date` class. Use a different variable name, like `myDate`, since it's an instance method.

Reply:
noted.